src/pages/DistributionsPage/DistributionsPage.test.tsx

```diff
@@ -147,7 +147,9 @@ test('render with no allocations', async () => {
 
   expect(screen.getByText('Gift Circle')).toBeInTheDocument();
   expect(
-    screen.getAllByText('Yearn USDC')[0].closest('button[role="combobox"]')
+    screen
+      .getAllByText('Yearn USDC 0xA0b8')[0]
```

Comment:
just in case this changes, you can also match a regex instead of a string: https://testing-library.com/docs/queries/about#textmatch